packages/tests/tests/peer_exchange.node.spec.ts

```diff
@@ -1,63 +1,68 @@
 import { waitForRemotePeer } from "@waku/core/lib/wait_for_remote_peer";
-import { createLightNode } from "@waku/create";
-import type { WakuLight } from "@waku/interfaces";
+import { createFullNode } from "@waku/create";
+import type { WakuFull } from "@waku/interfaces";
 import { Protocols } from "@waku/interfaces";
 
-import { makeLogFileName, Nwaku } from "../src";
+import { NOISE_KEY_1, Nwaku } from "../src";
 import { delay } from "../src/delay";
 
 describe("Peer Exchange: Node", () => {
-  let waku: WakuLight;
+  let waku: WakuFull;
   let nwaku1: Nwaku;
   let nwaku2: Nwaku;
   let nwaku3: Nwaku;
-
   afterEach(async function () {
     !!nwaku1 && nwaku1.stop();
     !!nwaku2 && nwaku2.stop();
     !!nwaku3 && nwaku3.stop();
+
     !!waku && waku.stop().catch((e) => console.log("Waku failed to stop", e));
   });
 
-  it("Queries successfully", async function () {
-    this.timeout(15_000);
+  it.only("Queries successfully", async function () {
```

Comment:
@danisharora099 I am guessing this should have been merged with `it.skip` instead, right?